opm/simulators/flow/BlackoilModel.hpp

```diff
@@ -971,19 +971,11 @@ namespace Opm {
             const bool relax_iter_cnv = (this->param_.min_strict_cnv_iter_ >= 0)
                 && (iteration >= this->param_.min_strict_cnv_iter_);
 
-            // Note trailing parentheses here, just before the final
-            // semicolon.  This is an immediately invoked function
-            // expression which calculates a single boolean value.
-            const auto relax_pv_fraction_cnv =
-                [&report, pvSum, numAquiferPvSum, this]()
-            {
-                const auto& cnvPvSplit = report.cnvPvSplit().first;
-
-                // [1]: tol < cnv <= relaxed
-                // [2]: relaxed < cnv
-                return static_cast<Scalar>(cnvPvSplit[1] + cnvPvSplit[2]) <
-                    this->param_.relaxed_max_pv_fraction_ * (pvSum - numAquiferPvSum);
-            }();
+            // [1]: tol < cnv <= relaxed
+            // [2]: relaxed < cnv
+            const auto& cnvPvSplit = report.cnvPvSplit().first;
+            const bool relax_pv_fraction_cnv = static_cast<Scalar>(cnvPvSplit[1] + cnvPvSplit[2]) <
+                this->param_.relaxed_max_pv_fraction_ * (pvSum - numAquiferPvSum);
```

Comment:
This introduces a new, unneeded, object (`cnvPvSplit`) into the scope.  I'd **really** prefer we use the original approach here.  Does CLang accept
```C++
[&report, this, eligible = pvSum - numAquiferPvSum]()
{
    // ...
    return static_cast<Scalar>(..) < fraction * eligible;
}();
```
instead?

Reply:
Yes this works, I will update the PR. Although I think the present version is more readable and therefore preferable despite introducing a extra alias...